src/components/Node/Node.tsx

```diff
@@ -33,86 +33,95 @@ const useStyles = makeStyles(
 );
 
 interface NodeProps {
+  _key: string;
   pathwayState: State;
   documentation: DocumentationResource | undefined;
   isOnPatientPath: boolean;
   isCurrentNode: boolean;
   xCoordinate: number;
   yCoordinate: number;
   expanded?: boolean;
-  onClickHandler?: () => void;
+  setExpanded: (key: string, expand?: boolean) => void;
+  interactive: boolean;
 }
 
-const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = forwardRef<HTMLDivElement, NodeProps>(
-  (
-    {
-      pathwayState,
-      documentation,
-      isOnPatientPath,
-      isCurrentNode,
-      xCoordinate,
-      yCoordinate,
-      expanded = false,
-      onClickHandler
-    },
-    ref
-  ) => {
-    const classes = useStyles();
-    const { label } = pathwayState;
-    const style = {
-      top: yCoordinate,
-      left: xCoordinate
-    };
-
-    const backgroundColorClass = isOnPatientPath
-      ? styles.onPatientPath
-      : clsx(styles.notOnPatientPath, classes['not-on-patient-path']);
-    const isActionable = isCurrentNode && !documentation;
-    const topLevelClasses = [styles.node, backgroundColorClass];
+const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = React.memo(
```

Comment:
since we typically import React functions explicitly (we don't use `React.useState`) I'd do that with memo too.

i.e. `import { memo} from 'react'`
and then just call `memo(...`